pkgs/applications/virtualization/x11docker/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, makeWrapper, nx-libs, xorg }:
+stdenv.mkDerivation rec {
+  name = "x11docker-${version}";
+  version = "5.4.1";
+  src = fetchFromGitHub {
+    owner = "mviereck";
+    repo = "x11docker";
+    rev = "v${version}";
+    sha256 = "0fcdr8i3crf4cina41h030q2jf5zvafll97iff129dl3sb27jnvi";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ nx-libs xorg.xhost xorg.xinit ];
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
prefer `dontBuild = true;` because it will enable all sort of niceties which come by default with the fixupPhase